openmetrics/datadog_checks/openmetrics/data/conf.yaml.example

```diff
@@ -94,6 +94,12 @@ instances:
     #
     # send_distribution_counts_as_monotonic: true
 
+    ## @param send_distribution_sums_as_monotonic - boolean - optional - default: false
+    ## Set send_distribution_sums_as_monotonic to true to send histograms & summary
+    ## sum as monotonic counters (instead of gauges).
+    #
+    # send_distribution_sums_as_monotonic: true
```

Comment:
:question: Should we also update configs for these?

- `amazon_msk`
- `cilium`
- `gitlab`

Asking because they also have `send_distribution_counts_as_monotonic` documented.

Reply:
I can update the other configs for prometheus based integrations as there are many missing both `send_distribution_counts_as_monotonic` and `send_distribution_sums_as_monotonic`